daemon/daemon_test.go

```diff
@@ -196,7 +196,7 @@ func TestContainerInitDNS(t *testing.T) {
 	hostConfig := `{"Binds":[],"ContainerIDFile":"","Memory":0,"MemorySwap":0,"CpuShares":0,"CpusetCpus":"",
 "Privileged":false,"PortBindings":{},"Links":null,"PublishAllPorts":false,"Dns":null,"DnsOptions":null,"DnsSearch":null,"ExtraHosts":null,"VolumesFrom":null,
 "Devices":[],"NetworkMode":"bridge","IpcMode":"","PidMode":"","CapAdd":null,"CapDrop":null,"RestartPolicy":{"Name":"no","MaximumRetryCount":0},
-"SecurityOpt":null,"ReadonlyRootfs":false,"Ulimits":null,"LogConfig":{"Type":"","Config":null},"CgroupParent":""}`
+"SecurityOpt":null,"ReadonlyRootfs":false,"Sysctls":null,"Ulimits":null,"LogConfig":{"Type":"","Config":null},"CgroupParent":""}`
```

Comment:
This struct should be marked with `omitempty`. The `null` value doesn't need to be sent if it is not there.